.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [8, 11, 17]
+        java: [11, 17]
```

Comment:
Will this impact any of the existing test runs against java8? `units-java8` job should cover the unit tests, but on fusion, I can see `java8-samples `as another job.

cc @thiagotnunes

Reply:
@suztomo We specify the docker image version for java8-samples here: https://github.com/googleapis/java-spanner/blob/main/.kokoro/nightly/java8-samples.cfg#L6

I think this will be alright, but double checking if we set the correct ENV variables in this container as well?

Reply:
@thiagotnunes  Thank you for the info. Yes, that line `  value: "gcr.io/cloud-devrel-kokoro-resources/java8"` matches the container image URL I have modified. It has `JAVA8_HOME` and `JAVA11_HOME` now. (I added the docker command invocation to my comment above)

Reply:
Thanks for the info LGTM